server/models/instance.coffee

```diff
@@ -33,4 +33,18 @@ CozyInstance.setResetKey = (key) ->
     # the key has a 1h TTL
     setTimeout ->
         resetKey = null
-    , 1 * 60 * 60 * 1000
\ No newline at end of file
+    , 1 * 60 * 60 * 1000
+
+domain = null
+CozyInstance.getDomain = (callback) ->
+    if domain?
+        return callback null, domain
```

Comment:
Do not use returns that break the function. See [Guidelines](https://github.com/cozy/cozy-guidelines#return-shortcuts).